contrib/init/datum_gateway.service

```diff
@@ -0,0 +1,89 @@
+[Unit]
+Description=DATUM Gateway
+Documentation=https://github.com/OCEAN-xyz/datum_gateway
+
+#
+# Expected file locations when using hardening options:
+# binary: /usr/bin/datum/
+# configuration: /etc/datum/
+# logs: /var/lib/datum_gateway/
+#
+
+# https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
+Wants=network-online.target
+After=network-online.target
+# alternatively, wait until bitcoind is up
+#After=bitcoind.service
+
+[Service]
+
+ExecStart=/usr/bin/datum/datum_gateway --config=/etc/datum/datum_gateway_config.json
+
+# Make sure the config directory is readable by the service user
+PermissionsStartOnly=true
+ExecStartPre=/bin/chgrp datum /etc/datum
+
+#
+# Process Management
+#
+
+Type=simple
+NotifyAccess=all
+PIDFile=/run/datum/datum_gateway.pid
+
+Restart=on-failure
+TimeoutStartSec=infinity
+TimeoutStopSec=600
+
+# limit number of open file descriptors
+LimitNOFILE=65535
+
+#
+# Directory Creation & Permissions
+#
+
+User=datum
+Group=datum
+
+# /run/datum_gateway
+RuntimeDirectory=datum_gateway
+RuntimeDirectoryMode=0710
+
+# /etc/datum
+ConfigurationDirectory=datum
+ConfigurationDirectoryMode=0710
+
+# /var/lib/datum_gateway
+StateDirectory=datum_gateway
```

Comment:
What is this for?

Reply:
A case of me following the bitcoind example too closely, I think. Removing this section and replacing with LogsDirectory pointed at /var/log/datum_gateway instead.